### DISCUSSION TITLE
切换为rsbuild后require(`xx/${a}.png`)图片动态加载方式失效了，要如何实现？

### DISCUSSION THREAD

**Discussion by robertyclin** [web-infra-dev/rsbuild#2504] [Q&A] [answered]
切换为rsbuild后require(`xx/${a}.png`)图片动态加载方式失效了，要如何实现？报[object Promise]

**Comment by chenjiahan** [comment, marked as answer]
Related issue: https://github.com/web-infra-dev/rsbuild/issues/2509
